lib/Common/Memory/HeapBucket.cpp

```diff
@@ -1617,6 +1623,9 @@ HeapBucketGroup<TBlockAttributes>::ScanInitialImplicitRoots(Recycler * recycler)
     smallFinalizableWithBarrierHeapBucket.ScanInitialImplicitRoots(recycler);
 #endif
     finalizableHeapBucket.ScanInitialImplicitRoots(recycler);
+#ifdef RECYCLER_VISITED_HOST
+    recyclerVisitedHostHeapBucket.ScanInitialImplicitRoots(recycler);
```

Comment:
Is this needed? Can visited host objects ever be implicit roots?

Reply:
I believe only the MemGC allocators set that bit, so this bucket should never contain any "implicit roots".  I'll try removing it.